### ISSUE TITLE
use .gitattributes for creating tarballs

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#637]
Follow-up to #560 which mentioned this but was mainly about something else. Cf, https://github.com/openSUSE/suse-xsl/blob/main/.gitattributes and https://github.com/openSUSE/suse-xsl/blob/main/Makefile#L151

This would obsolete packaging/exclude-files_for_daps_package.txt.

**Comment by tomschr** [comment]
I did the same for the geekodoc repo. However, I found out these things:

* The `.git` directory isn't included in the archive even though I haven't added it to `.gitattributes`. As a consequence, you don't need this line.
* You should ignore `.gitattributes` itself.
* It only has an effect if you call `git archive --worktree-attributes ...`

**Comment by ghost** [comment]
>     * The `.git` directory isn't included in the archive even though I haven't added it to `.gitattributes`. As a consequence, you don't need this line.

Thanks, feel free to fix that over in the other repo!


>     * It only has an effect if you call `git archive --worktree-attributes ...`

Not exactly. `--worktree-attributes` is only relevant when the `.gitattributes` file is not yet committed (or only an outdated version of it is committed).

from man git-attributes:
```
ATTRIBUTES
[....]
       Note that attributes are by default taken from the .gitattributes files
       in the tree that is being archived. If you want to tweak the way the
       output is generated after the fact (e.g. you committed without adding
       an appropriate export-ignore in its .gitattributes), adjust the checked
       out .gitattributes file as necessary and use --worktree-attributes
       option. Alternatively you can keep necessary attributes that should
       apply while archiving any tree in your $GIT_DIR/info/attributes file.
```

**Comment by fsundermeyer** [comment]
Was laready fixed with 8dfa08ae78fb07. Exclude files from packaging have been removed with 8301ac181.
